aten/src/ATen/native/native_functions.yaml

```diff
@@ -273,7 +273,7 @@
     CPU: _cosh_out_cpu
     CUDA: _cosh_out_cuda
 
-- func: cosine_embedding_loss(Tensor input1, Tensor input2, Tensor target, double margin=0.0, bool size_average=true, bool reduce=true) -> Tensor
+- func: cosine_embedding_loss(Tensor input1, Tensor input2, Tensor target, double margin=0.0, int64_t reduction=1) -> Tensor
```

Comment:
oh these magical numbers :/
can you include the enum header in `NativeFunctions.h` and use the enum here?

Reply:
I think I would have to changed the code gen to allow that. I will try it out.